test/e2e/run_networking_test.go

```diff
@@ -42,38 +42,43 @@ var _ = Describe("Podman run networking", func() {
 	})
 
 	It("podman run network connection with default bridge", func() {
-		session := podmanTest.Podman([]string{"run", "-dt", ALPINE, "wget", "www.podman.io"})
+		session := podmanTest.Podman([]string{"run", "-dt", fedoraMinimal, "curl", "-k", "-o", "/dev/null", "www.podman.io"})
 		session.Wait(90)
 		Expect(session).Should(Exit(0))
 	})
 
 	It("podman run network connection with host", func() {
-		session := podmanTest.Podman([]string{"run", "-dt", "--network", "host", ALPINE, "wget", "www.podman.io"})
+		session := podmanTest.Podman([]string{"run", "-dt", "--network", "host", fedoraMinimal, "curl", "-k", "-o", "/dev/null", "www.podman.io"})
 		session.Wait(90)
 		Expect(session).Should(Exit(0))
 	})
 
 	It("podman run network connection with default", func() {
-		session := podmanTest.Podman([]string{"run", "--network", "default", ALPINE, "wget", "www.podman.io"})
+		session := podmanTest.Podman([]string{"run", "--network", "default", fedoraMinimal, "curl", "-k", "-o", "/dev/null", "www.podman.io"})
 		session.WaitWithDefaultTimeout()
 		Expect(session).Should(Exit(0))
 	})
 
 	It("podman run network connection with none", func() {
-		session := podmanTest.Podman([]string{"run", "--network", "none", ALPINE, "wget", "www.podman.io"})
+		session := podmanTest.Podman([]string{"run", "--network", "none", fedoraMinimal, "curl", "-k", "-o", "/dev/null", "www.podman.io"})
```

Comment:
I can see the need for `curl` here, but all the other replacements seem excessive. And fragile. If you think they are absolutely necessary, would you consider refactoring them into a helper function with all the -k/-o/etc clutter in only one place? (And if they're not absolutely necessary, maybe revert them back to `wget`?)

Reply:
4 tests such as below are failed due to a proxy.
Therefore I think these tests should change from `wget` to `curl`.
I refactored these tests.

```
[Fail] Podman run networking [It] podman run network connection with default
/root/podman/test/e2e/run_networking_test.go:59

[Fail] Podman run networking [It] podman run network connection with none
/root/podman/test/e2e/run_networking_test.go:66

[Fail] Podman run networking [It] podman run network connection with private
/root/podman/test/e2e/run_networking_test.go:72

[Fail] Podman run networking [It] podman run network connection with loopback
/root/podman/test/e2e/run_networking_test.go:78
```